ingestr/src/sources.py

```diff
@@ -994,3 +995,42 @@ def dlt_source(self, uri: str, table: str, **kwargs):
         return readers(
             bucket_url=bucket_url, credentials=aws_credentials, file_glob=path_to_file
         ).with_resources(endpoint)
+
+
+class AsanaSource:
+    resources = [
+        "workspaces",
+        "projects",
+        "sections",
+        "tags",
+        "tasks",
+        "stories",
+        "teams",
+        "users",
+    ]
+
+    def handles_incrementality(self) -> bool:
+        return False
```

Comment:
it does handle incrementality, in the sense that the user cannot dictate it from the outside with flags like` --incremental-strategy`, even though the source says "we don't support incrementality".